test/classes/initializers/errors/multipleErrors.good

```diff
@@ -0,0 +1,5 @@
+multipleErrors.chpl:10: In initializer:
+multipleErrors.chpl:11: error: Field "x" used before it is initialized
+multipleErrors.chpl:11: error: Field "y" used before it is initialized
+multipleErrors.chpl:11: error: cannot call a method during phase 1 of initialization
+multipleErrors.chpl:11: error: cannot pass "this" to a function in phase 1 of initialization
```

Comment:
This is surprising to me.  I thought the proposal (at least in the chip) allows the object in phase 1 to be an instance of the parent class, and allows parent class method calls.  Shouldn't it also be passable-to-functions as an instance of the parent class? (``object`` in this case?)

Reply:
This error currently will apply to the old proposal. Eventually it will be updated to mention 'phase 0' or something about the region before super.init/this.init.

Reply:
I still feel confused.  This initializer includes an ``initDone()`` call, so it can't be the old proposal?  Unless there's a new new proposal already.

And chip 23, under "Simple classes" says if a ``super.init()`` is omitted, the compiler will insert one at the beginning of ``init()``.  So this ``printer()`` call will be in phase 1 in the new proposal.

Reply:
It's an artifact of the hybrid world on master as we transition towards the new initializer proposal. As you point out it's definitely a new initializer, but we don't currently have the 'behave like your parent' functionality implemented so I left the error message in (for now).